azure-pipelines.yml

```diff
@@ -119,7 +119,7 @@ jobs:
       failTaskOnFailedTests: true
 
 - job: BuildArtifacts  
-  condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/v2.x')))
+  condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/v2.x'), eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], true)))
```

Comment:
Do we need to add spaces for the variable UPLOAD_PACKAGE_TO_PRERELEASE_FEED?

Reply:
According to the help doc, we do not have to add spaces to the variable. Please see below:

![image](https://user-images.githubusercontent.com/12720858/119744582-2d792d80-be41-11eb-89e4-478295f55bc4.png)